packages/@aws-cdk/aws-ec2-alpha/lib/vpc-v2.ts

```diff
@@ -182,6 +191,62 @@ export interface VpcV2Props {
   readonly vpcName?: string;
 }
 
+/**
+ * Options to import a VPC created outside of CDK stack
+ */
+export interface VpcV2Attributes {
+
+  /**
+   * The VPC ID
+   * Refers to physical Id of the resource
+   */
+  readonly vpcId: string;
+
+  /**
+   * Arn of the VPC
+   * will be used to set value for account and region
+   * which then later can be used for establishing VPC peering connection
+   * @default - constructed with stack account and region value
+   */
+  readonly vpcArn?: string;
```

Comment:
IMOP, we either ask the customer to provide VPCId, or VPCArn .. I do not think the customer needs to provide both. Also, we need to let the customer when thy should provide the ID, and when they have to provide the Arn 

My assumption is  the arn is only required to support importing VPCs created in another region,  or account. But id should be used on all other scenarios.

Reply:
so in this case, do you think it is better to replace the ARN property with region, and account id?

Reply:
i would go with region and account id, instead of ARN.